features/features.go

```diff
@@ -47,7 +47,7 @@ var features = map[FeatureFlag]bool{
 	UseAIAIssuerURL:             false,
 	ReusePendingAuthz:           false,
 	CountCertificatesExact:      false,
-	IPv6First:                   false,
+	IPv6First:                   false, // deprecated
```

Comment:
Can this flag not be removed outright because staging/prod configs still include it?

Reply:
Correct.